ls1intum/Artemis
# Angular Strict Template for /app/exam/participate/summary/

### Motivation and Context



Angular strict template fixes

### Steps for Testing
1. Create an exam, then participate this exam with student credentials
2. Finish and submit the exam, go to exam summary page
3. Should behave as usual
